aspnetcore/native-aot/index.md

```diff
@@ -0,0 +1,190 @@
+---
+title: ASP.NET Core support for .NET native AOT
+author: mitchdenny
+description: Learn about ASP.NET Core support for .NET native AOT
+monikerRange: '>= aspnetcore-8.0'
+ms.author: midenn
+ms.custom: mvc
+ms.date: 04/10/2023
+uid: native-aot/index
+---
+# ASP.NET Core support for .NET native AOT.
+
+:::moniker range=">= aspnetcore-8.0"
+
+ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/). Not all ASP.NET Core features are compatible with native AOT at this time. This article shows the following for native AOT support in ASP.NET Core:
+* How to get started.
+* The benefits and potential drawbacks.
+* The limitations.
+
+## Getting started with .NET native AOT deployment in ASP.NET Core
+
+To help developers get started deploying with .NET native AOT in ASP.NET Core, use the:
```

Comment:
I think this section focuses too much on the api template. We need to introduce people to some basic concepts:
* Apps are developed normally
* Native AOT happens on publish
* Warnings alert you to possible AOT incompatibilities in your app.

Then, once we've introduced those basic concepts, walk people through using the api template.

Reply:
Do you mean just outline it like you have there in terms of the workflow? I'm thinking background information would be more completely covered here:

https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/

Is there a critical subset that we should cover here?

Reply:
I agree that we shouldn't try to repeat what is in the linked doc.

I think the bullet points above are the important things for people to know before they get started. Then a link to that doc for more information. Then go through using the api template as an example.